xhtml2pdf/context.py

```diff
@@ -163,18 +163,22 @@ def atFontFace(self, declarations):
         # The "src" attribute can be a CSS group but in that case
         # ignore everything except the font URI
         uri = data['src']
-        if not isinstance(data['src'], str):
+        fonts = []
+
+        if isinstance(data['src'], list):
             for part in uri:
-                if isinstance(part, str):
-                    uri = part
-                    break
-
-        src = self.c.getFile(uri, relative=self.c.cssParser.rootPath)
-        self.c.loadFont(
-            names,
-            src,
-            bold=bold,
-            italic=italic)
+                if isinstance(part, str) or isinstance(part, unicode):
```

Comment:
You can use the type basestring here instead which is exactly for these kinds of things.

Reply:
Nice, I just updated the PR.